packages/p2p/src/components/buy-sell/buy-sell-row.scss

```diff
@@ -90,6 +90,12 @@
 
     &__button {
         align-self: flex-start;
+        margin-left: auto;
+        height: fit-content;
+        min-height: 4rem;
+        width: fit-content;
+        white-space: inherit;
+        padding: 1rem 1.6rem;
```

Comment:
Added padding to cater when text is overflowing:
### Before:
<img width="458" alt="Screenshot 2022-07-18 at 11 27 21 AM" src="https://user-images.githubusercontent.com/103016120/179442584-ef539ecb-2c59-4b61-83c7-bebb8924de4a.png">

### After:
<img width="444" alt="Screenshot 2022-07-18 at 11 27 27 AM" src="https://user-images.githubusercontent.com/103016120/179442659-5e9e8295-9a24-4b38-b9ec-aa29be801636.png">

Reply:
Why not try with flex properties instead?

Reply:
i was also thinking that, wouldn't it be better with flex?